regression/contracts/variant_parsing_statement_fail/main.c

```diff
@@ -0,0 +1,14 @@
+
+int main()
+{
+  int i = 0; 
+  int N = 10;
+  while (i != N)
+    __CPROVER_loop_invariant(0 <= i && i <= N)
+    __CPROVER_decreases(int x = 0)
```

Comment:
Is it possible to have a `__CPROVER_decreases` without a loop invariant?

Reply:
Yes, it is possible. The default loop invariant is `true`. This code for that is already in my local repository, and I will file a second PR soon.